target/linux/ramips/dts/RBM11G.dts

```diff
@@ -0,0 +1,131 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "mikrotik,rbm11g", "mediatek,mt7621-soc";
+	model = "MikroTik RBM11G";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x10000000>;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		usr {
+			label = "rbm11g:green:usr";
+			gpios = <&gpio0 0 GPIO_ACTIVE_HIGH>;
+		};
+
+		rssi0 {
+			label = "rbm11g:green:rssi0";
+			gpios = <&gpio0 22 GPIO_ACTIVE_LOW>;
+		};
+
+		rssi1 {
+			label = "rbm11g:green:rssi1";
+			gpios = <&gpio0 23 GPIO_ACTIVE_LOW>;
+		};
+
+		rssi2 {
+			label = "rbm11g:green:rssi2";
+			gpios = <&gpio0 24 GPIO_ACTIVE_LOW>;
+		};
+
+		rssi3 {
+			label = "rbm11g:green:rssi3";
+			gpios = <&gpio0 25 GPIO_ACTIVE_LOW>;
+		};
+
+		rssi4 {
+			label = "rbm11g:green:rssi4";
+			gpios = <&gpio0 26 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+		res {
+			label = "res";
+			gpios = <&gpio0 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	pcie0_vcc_reg {
+		compatible = "regulator-fixed";
+		regulator-name = "pcie0_vcc";
+
+		regulator-min-microvolt = <3300000>;
+		regulator-max-microvolt = <3300000>;
+		gpio = <&gpio0 9 GPIO_ACTIVE_HIGH>;
+		enable-active-high;
+		regulator-boot-on;
+		regulator-always-on;
+	};
+};
+
+
+&spi0 {
+	status = "okay";
+
+	w25q128@0 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <3125000>;
+
+		partition@0 {
+			label = "routerboot";
+			reg = <0x000000 0x00F000>;
+			read-only;
+		};
+
+		factory: partition@f000 {
+			label = "factory";
+			reg = <0x00F000 0x031000>;
+			read-only;
+		};
+
+		partition@40000 {
+			label = "firmware";
+			reg = <0x040000 0xFC0000>;
+		};
+	};
+};
+
+&ethernet {
+	mtd-mac-address = <&factory 0x0010>;
+	mtd-mac-address-increment = <1>;
+};
+
+
+&pinctrl {
+	state_default: pinctrl0 {
+		gpio {
+			ralink,group = "uart2", "wdt", "sdhci";
```

Comment:
The rssi leds are using the pins of the rgmii2 group, which is at least missing. Please use https://www.kresin.me/files/pinmux_review_ramips.ods to check which pin groups need to be set to the gpio mode/function.

Reply:
Nice cheatsheet, thanks for sharing!